rimage/lazy_encoded.go

```diff
@@ -17,9 +20,9 @@ type LazyEncodedImage struct {
 	imgBytes []byte
 	mimeType string
 
-	decodeOnce   sync.Once
-	decodeErr    interface{}
-	decodedImage image.Image
+	decodeOnce     sync.Once
+	decodeImageErr interface{}
```

Comment:
why does this need to be an empty interface? why can't it just be an error?

Reply:
can we try changing it to an error?

Reply:
I'd rather not right now if that's okay - there's also a second error that is an interface in this struct - unclear why - and I'm unsure how much we want to go down the error refactoring rabbit hole.